src/chart/line/LineSeries.ts

```diff
@@ -52,6 +52,13 @@ interface LineStateOptionMixin {
     }
 }
 
+interface LineSegmentCallBackParams {
+    dataIndex: number,
+    dataVal: number
+}
+
+type LineSegmentOption = [number, number] | ((params: LineSegmentCallBackParams) => boolean);
```

Comment:
What does `segment: [1, 3]` mean? The 1st and the 3rd line or the lines between the 1st and the 3rd data?

Reply:
Represent the line segment to be selected with the index of the start point and end point.